src/app/containers/Toggle/useToggle.test.jsx

```diff
@@ -1,51 +1,119 @@
 /* eslint react/prop-types: 0 */
 import React from 'react';
-import { renderHook } from '@testing-library/react-hooks';
+import fetchMock from 'fetch-mock';
```

Comment:
Shall we add a test to make sure we are returned the default toggle value from the hook in the case that the API call fails?